core/src/main/java/org/apache/any23/extractor/rdf/BaseRDFExtractor.java

```diff
@@ -22,16 +22,20 @@
 import org.apache.any23.extractor.ExtractionParameters;
 import org.apache.any23.extractor.ExtractionResult;
 import org.apache.any23.extractor.Extractor;
-import org.eclipse.rdf4j.rio.RDFHandlerException;
-import org.eclipse.rdf4j.rio.RDFParseException;
-import org.eclipse.rdf4j.rio.RDFParser;
-import org.eclipse.rdf4j.rio.RioSetting;
+import org.eclipse.rdf4j.rio.*;
```

Comment:
Can you please make explicit imports rather than wildcard.

Reply:
Sure, sorry, that was something that IntelliJ did. I have no idea why. Will fix.